### ISSUE TITLE
serde_json_canonicalizer vs serde_jcs

### ISSUE THREAD

**Issue by silverpill** [evik42/serde-json-canonicalizer#1]
Hello,
Is there any practical difference between this library and [serde_jcs](https://github.com/l1h3r/serde_jcs)?

**Comment by evik42** [comment]
@silverpill Yes [serde_jcs](https://github.com/l1h3r/serde_jcs) seems to be abandoned and if you check the open issues there it is not fully compliant to the RFC in a few points:
- property sorting is not using the UTF-16 encoding
- number representation for large integers does not follow IEEE 754

I maintain this crate actively and it is listed on the [RFC reference repository](https://github.com/cyberphone/json-canonicalization/) as the Rust implementation.

**Comment by silverpill** [comment]
Thank you. I'll switch to your crate then.
